packages/react-components/react-skeleton/stories/Skeleton/SkeletonRow.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-skeleton';
+import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  invertedWrapper: {
+    backgroundColor: tokens.colorNeutralBackground1,
+  },
+  firstRow: {
+    alignItems: 'center',
+    display: 'grid',
+    paddingBottom: '10px',
+    position: 'relative',
+    ...shorthands.gap('5px'),
+    ...shorthands.margin('5px'),
+    gridTemplateColumns: 'min-content 80%',
+  },
+  secondRow: {
+    alignItems: 'center',
+    display: 'grid',
+    paddingBottom: '10px',
+    position: 'relative',
+    ...shorthands.gap('5px'),
+    ...shorthands.margin('5px'),
+    gridTemplateColumns: 'min-content 20% 20% 15% 15%',
+  },
+});
+
+const SkeletonElementsFirstRow = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.firstRow}>
+      <SkeletonItem shape="circle" size={24} />
+      <SkeletonItem shape="rectangle" size={16} />
+    </div>
+  );
+};
+
+const SkeletonElementsSecondRow = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.secondRow}>
+      <SkeletonItem shape="circle" size={24} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+    </div>
+  );
+};
+
+const SkeletonElementsThirdRow = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.secondRow}>
+      <SkeletonItem shape="square" size={24} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+    </div>
+  );
+};
+
+export const Row = (props: Partial<SkeletonProps>) => {
+  const styles = useStyles();
+  return (
+    <div className={styles.invertedWrapper}>
+      <Skeleton {...props}>
+        <SkeletonElementsFirstRow />
+        <SkeletonElementsSecondRow />
+        <SkeletonElementsThirdRow />
+      </Skeleton>
+    </div>
+  );
+};
```

Comment:
nit: having separate components for each row seems to overcomplicate this example IMO. It's fine if there is a specific reason to have separate components (maybe describe it in the story's description), but it might be easier to understand if it was all in a single component:
```ts
export const Row = (props: Partial<SkeletonProps>) => {
  const styles = useStyles();
  return (
    <div className={styles.invertedWrapper}>
      <Skeleton {...props}>
        <div className={classes.firstRow}>
          <SkeletonItem shape="circle" size={24} />
          <SkeletonItem shape="rectangle" size={16} />
        </div>
        <div className={classes.secondRow}>
          <SkeletonItem shape="circle" size={24} />
          <SkeletonItem size={16} />
          <SkeletonItem size={16} />
          <SkeletonItem size={16} />
         <SkeletonItem size={16} />
        </div>
        <div className={classes.secondRow}>
          <SkeletonItem shape="square" size={24} />
          <SkeletonItem size={16} />
          <SkeletonItem size={16} />
          <SkeletonItem size={16} />
          <SkeletonItem size={16} />
        </div>
      </Skeleton>
    </div>
  );
};
```